Tests/Runner/RunnerTest.php

```diff
@@ -2182,6 +2182,7 @@ public function testExecutorApplicationError(): void
                 ' failed: (2) Class 2 error',
                 $e->getMessage(),
             );
+            // @phpstan-ignore-next-line
```

Comment:
K tem `phpstan-ignore-next-line` by se hodil aspon kratky comment proc :pray:

Reply:
2185   Call to static method PHPUnit\Framework\Assert::assertCount() with 1 and array{} will always evaluate to
         false.
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting
            treatPhpDocTypesAsCertain: false in your phpstan.neon.